tests/Service/HelpServiceTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace App\Tests\Service;
+
+use App\Service\HelpService;
+
+class HelpServiceTest extends ServiceTestCase {
+    protected $helpService;
+
+    public function testGetDocumentTitlesList(): void
+    {
+        $this->helpService = static::getContainer()->get(HelpService::class);

```

Comment:
A couple minor things here:

1. We follow PSR-12 for code style, so the opening brace for a class should be on a new line. We have an automated check for this using PHP CS Fixer, but it looks like our configuration is only looking at the `src` directory, so our files currently aren't being checked. We should look into updating that sometime.
2. Since you're getting the HelpService from inside this test method, there's no reason to assign it to a class variable. I would suggest just setting it to a local variable to use in the test, or, if you prefer having it available in the class for other future tests, then you could add a `setUp` method and set it there, which will run before every test.